Mage/src/main/java/mage/game/permanent/PermanentImpl.java

```diff
@@ -146,7 +146,7 @@ public PermanentImpl(final PermanentImpl permanent) {
         this.markedLifelink = permanent.markedLifelink;
 
         for (Map.Entry<String, List<UUID>> entry : permanent.connectedCards.entrySet()) {
-            this.connectedCards.put(entry.getKey(), entry.getValue());
+            this.connectedCards.put(entry.getKey(), new ArrayList<>(entry.getValue()));
```

Comment:
@JayDi85 just confirming this is what you mean

Reply:
yes, that's right... list value must be independent between objects